README.md

```diff
@@ -17,20 +19,21 @@ This lib is replacement of [express-params](https://www.npmjs.com/package/expres
 var expressParams = require('express-params-handler')
 
 var app = express()
-expressParams(app)('id', Number)
-expressParams(app)('date', /^\d{4}-\d{2}-\d(2)$/)
 
-app.get('/:id', function(req, res, next) {
+app.param('id', expressParams(Number))
+app.param('date', expressParams(/^\d{4}-\d{2}-\d{2}$/))
+
+app.get('/by-id/:id', function(req, res, next) {
```

Comment:
In real app I will use `/:id` and `/by-date/:date`. It is not really important. So `/by-id/:id` is good